experimental/bits/simd_builtin.h

```diff
@@ -624,7 +624,7 @@ __convert_all(_From __v)
 	    return __vector_bitcast<_FromT, decltype(__n)::value>(__vv);
 	  };
 	  [[maybe_unused]] const auto __vi = __to_intrin(__v);
-	  auto&& __make_array = [](std::initializer_list<auto> __xs) {
+	  auto&& __make_array = [](auto __xs) {
```

Comment:
This can't work. An initializer list argument cannot be deduced as one. But I have a fix coming up for this. Erich mentioned it yesterday.

Reply:
Yes, we talked with Erich yesterday about that.